Validation/RecoTrack/scripts/harvestTrackValidationPlots.py

```diff
@@ -31,7 +31,7 @@
     os.chdir(_tempdir)
 
     # compile cmsDriver command
-    cmsDriverCommand = "cmsDriver.py harvest --scenario pp --filetype DQM --conditions auto:run2_mc --mc -s HARVESTING:postProcessorTrackSequence -n -1 --filein {0}".format(filelist)
+    cmsDriverCommand = "cmsDriver.py harvest --scenario pp --filetype DQM --conditions auto:run2_mc --mc -s HARVESTING:@trackingOnlyValidation -n -1 --filein {0}".format(filelist)
```

Comment:
@lveldere I decided to go with `@trackingOnlyValidation` auto-sequence since it is a bit more general than `postProcessorTrackSequenceTrackingOnly` (and thus has a bit higher probability to stay as the most inclusive sequence).